### ISSUE TITLE
ColumnProcessor is not giving proper output with CsvParserSettings.selectIndexes() 

### ISSUE THREAD

**Issue by sagardesai87** [uniVocity/univocity-parsers#183]
I want to select only two columns data from [example.txt](https://github.com/uniVocity/univocity-parsers/files/1236521/example.txt) using code below. But its giving all columns data in output with improper structure.
`String csvPath = "D:/example.csv";`
		`CsvParserSettings csvParserSettings = new CsvParserSettings();`
		`ColumnProcessor rowProcessor = new ColumnProcessor();`
		`csvParserSettings.setProcessor(rowProcessor);`
		`csvParserSettings.selectIndexes(1,2);`
		`csvParserSettings.setColumnReorderingEnabled(false);`
		`csvParserSettings.getFormat().setLineSeparator("\n");`
		`csvParserSettings.setHeaderExtractionEnabled(false);`
		`csvParserSettings.setHeaders("");`
		`CsvParser csvparser = new CsvParser(csvParserSettings);`
		`csvparser.parse(parser.getReader(csvPath));`
		`System.out.println(rowProcessor.getColumnValuesAsMapOfIndexes());`

Below is the output of above code snippet:

{0=[null, null, null, null, null], 
1=[R0C1, R0C1, R2C1, R3C1, R4C1], 
2=[R0C2, R0C2, R2C2, R3C2, R4C2], 
3=[null, R0C3, R2C3, R3C3, R4C3], 
4=[null, R0C4, R2C4, R3C4, R4C4], 
5=[null, R0C5, R2C5, R3C5, R4C5], 
6=[null, null, null, null], 
7=[R1C1, null, null, null], 
8=[R1C2, null, null, null], 
9=[R1C3, null, null, null], 
10=[R1C4, null, null, null], 
11=[R1C5, null, null, null]}

**Comment by jbax** [comment]
Thank you! 

You will get the result you expect if you remove the following lines from your code:

```
// csvParserSettings.setHeaders("");

```

This means the input has one single header: an empty string. The column processor apparently goes crazy if you do that, I just fixed the problem.

And

```
// csvParserSettings.setColumnReorderingEnabled(false); 
```

This means you want all columns, including the ones you didn't select. The columns not selected will be filled with nulls.

Regardless, the correct output given the configuration you provided should be a map with 6 columns (from 0 to 5, with 0, 3, 4 and 5 containing nulls only) so there's a bug here.

**Comment by jbax** [comment]
Fixed and released a 2.5.4-SNAPSHOT build that includes the adjustments required to make everything behave as expected. Thank you for using our parsers!
